cogs/member_stats.py

```diff
@@ -1,21 +1,82 @@
 import discord
-from discord.ext import commands
+from discord.ext import tasks, commands
 from pathlib import Path
 from .validation import is_moderator, is_mod_commands_channel
 import json
+import os
+from os import path
+from datetime import datetime
+from dateutil import tz
```

Comment:
If you're up for it, you might like to try adding these new dependencies to `poetry`.

Reply:
I just remembered that `datetime` is already in the standard library. So just `dateutil` would need to be added.